pkg/util/cloud_prepper.go

```diff
@@ -17,31 +17,26 @@ limitations under the License.
 package util
 
 import (
-	"regexp"
-
 	"github.com/containers/image/docker"
+	"github.com/docker/docker/client"
 )
 
 // CloudPrepper prepares images sourced from a Cloud registry
 type CloudPrepper struct {
-	ImagePrepper
+	Source string
+	Client *client.Client
 }
 
 func (p CloudPrepper) Name() string {
 	return "Cloud Registry"
 }
 
 func (p CloudPrepper) GetSource() string {
-	return p.ImagePrepper.Source
+	return p.Source
 }
 
-func (p CloudPrepper) SupportsImage() bool {
-	pattern := regexp.MustCompile("^.+/.+(:.+){0,1}$")
-	image := p.ImagePrepper.Source
-	if exp := pattern.FindString(image); exp != image || CheckTar(image) {
-		return false
-	}
-	return true
+func (p CloudPrepper) GetImage() (Image, error) {
+	return getImage(p)
```

Comment:
is getImage used anywhere else? Should you just inline it here instead of breaking it out?

Reply:
yeah it's actually the same across all the preppers (currently), but it made more sense to me to just have them all use a helper method and expose the `GetImage()` publicly on each of the preppers. WDYT?